source/SIL.AppBuilder.Portal.Frontend/src/data/containers/with-current-user/hoc.tsx

```diff
@@ -143,52 +143,74 @@ export function withCurrentUser(opts = {}) {
 
           await pushPayload(updateStore, json);
 
-          // this state value isn't used anywhere, but we need to trigger
-          // a re-render once the current user data has finished being consumed
-          this.makingRequest = false;
-          this.setState({ networkFetchComplete: true });
+          const usersFromCache = await dataStore.cache.query(cacheQuery());
+          const currentFromCache = usersFromCache[0];
+
+          if (currentFromCache) {
+            this.makingRequest = false;
```

Comment:
since the final catch calls set state... should probably set `this.makingRequest = false` after that catch, right now makingRequest doesn't get reset if there is an error response. ( or it doesn't look like it does ).

Reply:
good _catch_